File: app/models/place.rb
class Place < ApplicationRecord
  has_many :users, inverse_of: :place
  has_many :bills, inverse_of: :place, dependent: :destroy
  # has_many :members, class_name: 'User', foreign_key: 'id'
  belongs_to :payer, class_name: 'User', foreign_key: 'payer_id', required: true
end

File: frontend/app/models/place.js
import DS from 'ember-data';

const { attr, belongsTo, hasMany } = DS;

export default DS.Model.extend({
  name: attr('string'),
  description: attr('string'),
  address: attr('string'),
  area: attr('number'),
  capacity: attr('number'),
  rent: attr('number'),
  users: hasMany('user'),
  payer: belongsTo('user', { inverse: 'place' }),
  bills: hasMany('bill')
});
